ambry-protocol/src/main/java/com.github.ambry.protocol/AmbryRequests.java

```diff
@@ -167,15 +125,14 @@ public void handleRequests(Request request) throws InterruptedException {
         case DeleteRequest:
           handleDeleteRequest(request);
           break;
+        case TtlUpdateRequest:
+          handleTtlUpdateRequest(request);
+          break;
         case ReplicaMetadataRequest:
           handleReplicaMetadataRequest(request);
           break;
         case AdminRequest:
           handleAdminRequest(request);
```

Comment:
are we missing a break statement here for AdminRequest case?

Reply:
good catch, added.